x/leverage/simulation/operations.go

```diff
@@ -143,12 +143,12 @@ func SimulateMsgWithdrawAsset(ak simulation.AccountKeeper, bk types.BankKeeper,
 		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
 		accs []simtypes.Account, chainID string,
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
-		from, coin, skip := randomCollateralFields(r, ctx, accs, lk)
+		from, uToken, skip := randomCollateralFields(r, ctx, accs, lk)
```

Comment:
There are two places to find coins eligible for withdrawal
1) A user's collateral-enabled uTokens, which are searched here using `GetBorrowerCollateral`
2) A user's collateral-_disabled_ uTokens, which are part of their `SpendableCoins`. In order to find only uTokens in `SpendableCoins` and ignore tokens, you must look for denoms that start with `"u/"`

In order to simulate withdrawals of both types, a function `randomWithdrawFields` might be created which collects both types of uTokens eligible for withdrawal into a single `sdk.Coins`, and returns `simtypes.RandSubsetCoins` from that.

Reply:
done at https://github.com/umee-network/umee/pull/433/commits/2a4fdb2d124bb3c326aa5e5f8593b7967b8fee2b